api/src/main/kotlin/dexportal/routes/mms/OrganizationDataStreamRoutes.kt

```diff
@@ -0,0 +1,13 @@
+package dexportal.routes.mms
+
+import io.ktor.server.routing.*
+
+fun Route.organizationDataStreamsRouting() {
+
+    route("/organizations") {
+        get("{organizationId}/datastreams") { }
```

Comment:
Pick one placeholder name for consistency with other organization routes.  `orgId` or `organizationId`.

Reply:
Good call, I grabbed these directly from MMS, so we would need to update the consistency in MMS. I'll update here. Let's go with the abbreviated version in both MMS and Portal.